src/main/java/org/folio/linked/data/configuration/kafka/KafkaProducerConfiguration.java

```diff
@@ -29,7 +29,7 @@ public class KafkaProducerConfiguration {
   private final LinkedDataTopicProperties linkedDataTopicProperties;
 
   @Bean
-  public FolioMessageProducer<ResourceIndexEvent> resourceIndexEventProducer(
+  public FolioMessageProducer<ResourceIndexEvent> indexEventProducer(
```

Comment:
Minor -> It will be nice to rename this bean to `bibliographicIndexEventProducer` (or `bibIndexEventProducer`) to be consistent with the bean `authorityIndexEventProducer`.